addons/mail/static/src/new/composer/composer.js

```diff
@@ -27,6 +27,7 @@ import { FileUploader } from "@web/views/fields/file_handler";
 import { Typing } from "./typing";
 import { useDebounced } from "@web/core/utils/timing";
 import { browser } from "@web/core/browser/browser";
+import { useSuggestion } from "./composer_hook";
```

Comment:
I don't like this file name, it should be more like suggestion hook, except it is not generic enough.

Reply:
It's intentional to keep this hook limited to composer for the moment.
As I said, highest priority is to reduce complexity of `Composer` component, and encapsulating the feature in a hook does precisely contributes to that objective.

I will see at improving the hook at a later date